.github/workflows/tmp-p114-canary-network-strengthen.yml

```
@@ -0,0 +1,398 @@
+          )
+          p['proofGate'] = (
+              'Representative multi-response sequences keep the Canary compact and semantically stable; WAB, Guest, Hardwire, Local, and Arbitrary/N/A '
+              'are preserved as exact required-network labels; Arbitrary/N/A is never used to hide an unknown requirement; a seeded omission or wrong '
+              'profile/network is detected and recoverable from authoritative context; unknown identity or network requirement is never fabricated; '
+              'stylistic variation is not falsely treated as context exhaustion; explicit validation order covers stable baseline, recovery, legitimate change, '
+              'repeated drift, then unrecoverable state; repeated drift crosses the handoff threshold; and the handoff preserves the current task, proof floor, '
+              'forbidden scope, required network, and first executable continuation without duplicating full closeout or context-factoring owners.'
+          )
+          p['copyContent'] = r'''RUN A LIGHTWEIGHT CONVERSATION CANARY THROUGHOUT THE CHAT. DO NOT TURN IT INTO A SECOND TASK REPORT, A GIANT HEADER, OR A TOKEN COUNTER.
+
+CANARY ANCHOR
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)